DNN Platform/Library/Security/PortalSecurity.cs

```diff
@@ -2,1001 +2,986 @@
+            new Regex("<form[^>]*>.*?</form[^><]*>", RxOptions),
+            new Regex("<option[^>]*>.*?</option[^><]*>", RxOptions),
+            new Regex("<select[^>]*>.*?</select[^><]*>", RxOptions),
+            new Regex("<source[^>]*>.*?</source[^><]*>", RxOptions),
+            new Regex("<iframe[^>]*>.*?</iframe[^><]*>", RxOptions),
+            new Regex("<iframe.*?<", RxOptions),
+            new Regex("<iframe.*?", RxOptions),
+            new Regex("<ilayer[^>]*>.*?</ilayer[^><]*>", RxOptions),
+            new Regex("<form[^>]*>", RxOptions),
+            new Regex("</form[^><]*>", RxOptions),
+            new Regex("\bonerror\b", RxOptions),
+            new Regex("\bonload\b", RxOptions),
+            new Regex("\bonfocus\b", RxOptions),
+            new Regex("\bonblur\b", RxOptions),
+            new Regex("\bonclick\b", RxOptions),
+            new Regex("\bondblclick\b", RxOptions),
+            new Regex("\bonchange\b", RxOptions),
+            new Regex("\bonselect\b", RxOptions),
+            new Regex("\bonsubmit\b", RxOptions),
+            new Regex("\bonreset\b", RxOptions),
+            new Regex("\bonkeydown\b", RxOptions),
+            new Regex("\bonkeyup\b", RxOptions),
+            new Regex("\bonkeypress\b", RxOptions),
+            new Regex("\bonmousedown\b", RxOptions),
+            new Regex("\bonmousemove\b", RxOptions),
+            new Regex("\bonmouseout\b", RxOptions),
+            new Regex("\bonmouseover\b", RxOptions),
+            new Regex("\bonmouseup\b", RxOptions),
+            new Regex("\bonreadystatechange\b", RxOptions),
+            new Regex("\bonfinish\b", RxOptions),
+            new Regex("javascript:", RxOptions),
+            new Regex("vbscript:", RxOptions),
+            new Regex("unescape", RxOptions),
+            new Regex("alert[\\s(&nbsp;)]*\\([\\s(&nbsp;)]*'?[\\s(&nbsp;)]*[\"(&quot;)]?", RxOptions),
+            new Regex(@"eval*.\(", RxOptions),
+        };
+
+        private static readonly Regex DangerElementsRegex = new Regex(@"(<[^>]*?) on.*?\=(['""]*)[\s\S]*?(\2)( *)([^>]*?>)", RxOptions);
+        private static readonly Regex DangerElementContentRegex = new Regex(@"on.*?\=(['""]*)[\s\S]*?(\1)( *)", RxOptions);
+
+        /// -----------------------------------------------------------------------------
+        /// <summary>
+        /// The FilterFlag enum determines which filters are applied by the InputFilter
+        /// function.  The Flags attribute allows the user to include multiple
+        /// enumerated values in a single variable by OR'ing the individual values
+        /// together.
+        /// </summary>
+        /// -----------------------------------------------------------------------------
+        [Flags]
+        public enum FilterFlag
+        {
+            MultiLine = 1,
+            [Obsolete("Deprecated in 9.8.1. Scheduled removal in V11.0.0. A direct call to HttpUtility.HtmlEncode should be used")]

```

Comment:
I think we should be recommending `System.Net.WebUtility` over `System.Web.HttpUtility`, since it's included in .NET Standard.
```suggestion
            [Obsolete("Deprecated in 9.8.1. Scheduled removal in v11.0.0. A direct call to WebUtility.HtmlEncode should be used")]
```